db/schema.sql

```diff
@@ -1,12 +1,10 @@
 create sequence seq_pst_id increment 1 start 1;
 create table preset_states (
-  pst_id                    bigint not null default nextval('seq_pst_id'),
```

Comment:
AFAIK primary key is required

Reply:
Column id is not used, so is not required. We don't have relations here, imo primary key is not required.

Reply:
well but you can lose a performance, WHERE and JOINS are much faster on tables with a index.